apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/OrdersTableContent.tsx

```
@@ -37,11 +37,16 @@ export function OrdersTableContent({
     return <UnsupportedNetworkContent />
   }
 
+  if (isProviderNetworkUnsupported) {
+    return <UnsupportedNetworkContent />
+  }
+
   return filteredOrders?.length === 0 ? (
     <NoOrdersContent
       currentTab={currentTab}
       searchTerm={searchTerm}
       historyStatusFilter={historyStatusFilter}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)